posts/2022-08-24-open-liberty-with-cargo-tracker.adoc

```diff
@@ -0,0 +1,106 @@
+---
+= Try out Cargo Tracker, an enterprise-level Jakarta EE application, with Open Liberty
+Chanun Ong <https://github.com/ocseal>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+//Blank line here is necessary before starting the body of the post.
+
+As one of the top Java framework choices for enterprise-level applications, Jakarta EE can be found at the center of projects all over the globe. The https://eclipse-ee4j.github.io/cargotracker/[Cargo Tracker] open-source application was designed to replicate a complex enterprise application by utilizing core Jakarta EE features with professional development standards. Now, you can get a taste of Open Liberty in an industry environment by running Cargo Tracker with an Open Liberty server all on your own computer.
+
+== The Cargo Tracker application
+
+Cargo Tracker simulates an industry-developed freight tracking web application built with Jakarta EE 8 and domain-driven design principles. The application uses multiple key Jakarta EE features, including Faces, CDI, enterprise beans, bean validation, REST, and JSON binding. Persistence is also implemented along with batching and JMS messaging. 
+
+Functionally, the application is set up to book, track, update, and route cargo objects. Messaging events are logged to update cargo locations while persistence ensures that cargo status is not changed or removed when the application is shut down. Each portion of Cargo Tracker represents a step in a real-life shipment process: the customer tracks their cargo using the Public Tracking interface, the shipping company schedules and routes deliveries using the Administration interface, and handlers in ports log arriving and departing cargo using the Mobile Event Logger. 
+
+
+image::/img/blog/cargo-tracker-home.png[Cargo Tracker Home Page,width=70%,align="center"]
+
+
+Cargo Tracker was originally developed to work with the Payara runtime and had not been previously used with Open Liberty, but the application's functionality was perfectly preserved after the server switch. There were no issues across operating systems, as Ubuntu 18.04, Windows 10, and Mac OS Monterey 12.4 were all able to run Cargo Tracker identically. 
+
+
+image::/img/blog/cargo-tracker-map.png[Cargo Tracker Map,width=70%,align="center"]
+
+== Running Cargo Tracker with Open Liberty yourself
+
+Feel free to give Cargo Tracker with Open Liberty a try on your own! Cargo Tracker supports Java SE 8, 11, and 17 — https://developer.ibm.com/languages/java/semeru-runtimes/downloads[IBM Semeru Runtime] is preferred when using Open Liberty. Before cloning the application to your machine, install any required JDKs and make sure that your JAVA_HOME is compatible. To access the repository remotely you'll need to install https://git-scm.com/book/en/v2/Getting-Started-Installing-Git[Git] if you haven't already. Finally, you will need to install https://maven.apache.org/download.cgi[Maven] as it is required to build and run the application. 
```

Comment:
"make sure that your JAVA_HOME is compatible"

Do you mean "export JAVA_HOME=...."  to the location?  Or what did you mean?